src/store/home/home.thunks.ts

```diff
@@ -208,16 +215,18 @@ function refreshAccounts(
  * @returns {void}
  */
 function fetchPoolTransactions() {
-  return (dispatch, getState) => {
+  return (dispatch: AppDispatch, getState: () => RootState) => {
     dispatch(homeActions.loadPoolTransactions());
 
     const {
       global: { wallet },
     } = getState();
 
-    getPoolTransactions(null, wallet.publicKeyCompressedHex)
-      .then((transactions) => dispatch(homeActions.loadPoolTransactionsSuccess(transactions)))
-      .catch((err) => dispatch(homeActions.loadPoolTransactionsFailure(err)));
+    if (wallet !== undefined) {
+      getPoolTransactions(undefined, wallet.publicKeyCompressedHex)
+        .then((transactions) => dispatch(homeActions.loadPoolTransactionsSuccess(transactions)))
+        .catch((err) => dispatch(homeActions.loadPoolTransactionsFailure(err)));
+    }
```

Comment:
Should we dispatch a `loadPoolTransactionsFailure` if the wallet is undefined?

Reply:
We can discuss this in the meet as well, I've some doubts about the `wallet` also (we depend in the wallet a lot of times across the entire app).

Reply:
I think that if `wallet === undefined` we can just do nothing. In that case we shouldn't be able to arrive in this view and dispatch this thunks.